src/style.css

```diff
@@ -0,0 +1,85 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+html {
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+}
+
+.form {
+  position: relative;
+  margin: 20px auto;
+}
+
+.first__input {
```

Comment:
The class shouldn’t depend on HTML structure, if I swap input places should I rename this class? Let's use big and small